apis/management.cattle.io/v3/k8s_defaults.go

```diff
@@ -19,6 +19,7 @@ var (
 		"v1.9.7-rancher2-2",
 		"v1.10.5-rancher1-2",
 		"v1.11.1-rancher1-1",
```

Comment:
we should remove this one. there should only ever be 1 version per minor version of k8s